tests/test_github.py

```diff
@@ -840,7 +807,7 @@ def test_github_checkout_prs(self) -> None:
     @mock.patch('git_machete.github.GITHUB_REMOTE_PATTERNS', FAKE_GITHUB_REMOTE_PATTERNS)
     @mock.patch('urllib.request.urlopen', MockContextManager)
     @mock.patch('urllib.request.Request', git_api_state_for_test_github_checkout_prs_fresh_repo.new_request())
-    def test_github_checkout_prs_freshly_cloned(self) -> None:
+    def test_github_checkout_prs_freshly_cloned(self, tmp_path: Any) -> None:
```

Comment:
Would it make sense to type these test case params more specifically as e.g. `str`?